helix-core/src/main/java/org/apache/helix/controller/rebalancer/util/DelayedRebalanceUtil.java

```diff
@@ -128,6 +129,14 @@ private static Set<String> getActiveNodes(Set<String> allNodes, Set<String> live
     return activeNodes;
   }
 
+  private static Set<String> filterEvacuatingInstances(Map<String, InstanceConfig> instanceConfigMap,
```

Comment:
Would recommend renaming to `filterOutEvacuatingInstances`, as `filterEvacuatingInstances` sounds like it should be returning the EVACUATE nodes